chore: add changelog for #9986 (#10018)
emqx/emqx: changes/ce/feat-9986.en.md

> Add MQTT ingress and remove mgmt references in helm charts

Reviewer: ```suggestion
For helm charts, add MQTT ingress bridge; and removed stale `mgmt` references.
```
